vulnerabilities/models.py

```diff
@@ -281,3 +317,22 @@ class VulnerabilitySeverity(models.Model):
 
     class Meta:
         unique_together = ("vulnerability", "reference", "scoring_system")
+
+
+class Advisory(models.Model):
+    """
+    An advisory directly obtained from upstream without any modifications.
+    """
+
+    date_published = models.DateField(help_text="UTC Date of publication of the advisory")
+    date_collected = models.DateField(help_text="UTC Date on which the advisory was collected")
+    source = models.CharField(
```

Comment:
source may not be the best name. See also above another comments wrt. source naming https://github.com/nexB/vulnerablecode/pull/525/files#diff-7f9f2c92e7163b06d21fa139369d75caed6561d0368b60ea2516cece0220eb5bR201

Reply:
renamed to `created_by`